pkg/cmd/cli/install/install.go

```diff
@@ -237,3 +257,41 @@ func (o *InstallOptions) Validate(c *cobra.Command, args []string, f client.Fact
 
 	return nil
 }
+
+// parseResourceRequests takes a set of CPU and memory requests and limit string
+// values and returns a ResourceRequirements struct to be used in a Container.
+// An error is returned if we cannot parse the request/limit.
+func parseResourceRequests(cpuRequest, memRequest, cpuLimit, memLimit string) (corev1.ResourceRequirements, error) {
```

Comment:
it might make sense to move this to a helper package, though I'm not sure where it could go